ext/intl/resourcebundle/resourcebundle.stub.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class ResourceBundle implements Traversable
+{
+    public function __construct(?string $locale, ?string $bundlename, bool $fallback = true) {}
+
+    /** @return ResourceBundle|false|null */
+    public static function create(?string $locale, ?string $bundlename, bool $fallback = true) {}
+
+    /**
+     * @param string|int $index
+     * @return mixed
+     */
+    public function get($index, bool $fallback = true) {}
+
+    /** @return int|false */
+    public function count() {}
+
+    /** @return array|false */
+    public static function getLocales(string $bundlename) {}
+
+    /** @return int */
+    public function getErrorCode() {}
+
+    /** @return string */
+    public function getErrorMessage() {}
+}
+
+/** @return ResourceBundle|false|null */
+function resourcebundle_create(?string $locale, ?string $bundlename, bool $fallback = true) {}
+
+/**
+ * @param string|int $index
+ * @return mixed
+ */
+function resourcebundle_get(ResourceBundle $bundle, $index) {}
+
+function resourcebundle_count(ResourceBundle $bundle): int|false {}
```

Comment:
We should convert the "unconstructed" case into a hard Error exception in the future.

Reply:
I'd like this change! Will provide a PR sometime soon.